src/installer/image/installer.go

```diff
@@ -44,15 +46,15 @@ func GetDigest(uri string) (string, error) {
 	return canonRef.Digest().Encoded(), nil
 }
 
-func NewImageInstaller(fs afero.Fs, props *Properties) installer.Installer {
+func NewImageInstaller(fs afero.Fs, props *Properties, apiReader client.Reader, pullSecret v1.Secret) (installer.Installer, error) {
```

Comment:
```suggestion
func NewImageInstaller(fs afero.Fs, apiReader client.Reader, props *Properties, pullSecret v1.Secret) (installer.Installer, error) {
```

(also `Properties` already has an `ApiReader`)

Reply:
@0sewa0 I saw you mentioned it in another PRs: let's add it to CONTRIBUTING.md -> code-styles

Reply:
created PR: https://github.com/Dynatrace/dynatrace-operator/pull/2058